pnpm-workspace.yaml

```diff
@@ -0,0 +1,2 @@
+- packages
+  - playground
```

Comment:
```suggestion
packages:
  - playground
```

Reply:
@antfy looks like https://github.com/antfu/unplugin-vue2-script-setup/blob/main/pnpm-workspace.yaml is using the wrong syntax?

https://pnpm.io/pnpm-workspace_yaml